test/upgrade/prober/verify.go

```diff
@@ -101,7 +102,11 @@ func (p *prober) Verify() (eventErrs []error, eventsSent int) {
 		} else if p.config.OnDuplicate == Error {
 			eventErrs = append(eventErrs, errors.New(t))
 		}
-		p.exportStepEventTrace(i, t)
+		if strings.HasPrefix(t, "finish event") {
```

Comment:
Can we extract a constant for this string and use it in both call and verification sites?

Reply:
OK

Reply:
Pushed.